app/models/user.rb

```diff
@@ -21,22 +21,26 @@ class User < ActiveRecord::Base
 
   # Scopes #####################################################################
 
-  scope :active,   -> { where("status = 'active'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
-  scope :alumni,   -> { where("status = 'alumni'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
-  scope :pending,  -> { where("status = 'pending'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
-  scope :inactive, -> { where("status = 'inactive'").
-                        sort_by { |u| [u.last_name, u.first_name] } }
+  scope :active,          -> { where("status = 'active'").
+                               sort_by { |u| [u.last_name, u.first_name] } }
+  scope :alumni,          -> { where("status = 'alumni'").
+                               sort_by { |u| [u.last_name, u.first_name] } }
+  scope :pending,         -> { where("status = 'pending'").
+                               sort_by { |u| [u.last_name, u.first_name] } }
+  scope :inactive,        -> { where("status = 'inactive'").
+                               sort_by { |u| [u.last_name, u.first_name] } }
```

Comment:
Expression at 31, 76 should be on its own line.<br>Use 2 (not 29) spaces for indenting an expression spanning multiple lines.